src/platform/forms-system/test/js/definitions/profileContactInfo.unit.spec.jsx

```diff
@@ -1,6 +1,8 @@
 import { expect } from 'chai';
 
-import profileContactInfo from '../../../src/js/definitions/profileContactInfo';
+import profileContactInfo, {
+  profileReviewErrorOverride,
```

Comment:
is the comma needed inside the ` {     } `

Reply:
Yeah, eslint would complain about it if it wasn't there. It enforces trailing commas everywhere